### ISSUE TITLE
Adjustable Mandatory Hashtag

### ISSUE THREAD

**Issue by adiatmad** [hotosm/tasking-manager#6476]
This is come from feedback in [Project 16807](https://tasks.hotosm.org/projects/16807)

1. **Is your feature request related to a problem? Please describe.**

The hashtags are primarily used for tracking and filtering, so it's not helpful for someone reviewing change-sets to see them each time. Additionally, if the comment is lengthy, only the first few characters might be visible. Reviewers are generally interested in the manually written comment, so it should be placed at the beginning.

2. **Describe the solution you'd like**

Please enable project managers to move the mandatory hashtag (_hotosm-project-xxxx_) to the end of the comment. This will make it easier for reviewers to read while keep it trackable for monitoring.
<img width="441" alt="Required hashtag" src="https://github.com/hotosm/tasking-manager/assets/38268616/bc1bb05b-b2ab-433b-8909-81f2e8d3af73">

**Comment by martien-176** [comment]
Can't the mapper place that comment before the hashtag themselves, manually?

**Comment by ramyaragupathy** [comment]
I agree with @martien-176 for this being at the editor level - we can see how to pick this up integrated iD editor.

**Comment by cliffordsnow** [comment]
I'm with DWG and often get tickets involving tasking manager edits. Ideally the task would have a brief description of the project and then the hashtags. It would really make it much easier. Now I have to go to the task in the tasking manger to learn what is supposed to be happening. Asking the user to manually add a description is likely not going to happen.

**Comment by ramyaragupathy** [comment]
@prabinoid @suzit-10 -  how about utilising the short description field for changeset comments and then populating the default hashtags? Please let me know if there are implementation challenges.

**Comment by Adrian-Shobrooke** [comment]
Can we use the project features to be mapped to add to the default changeset comment? (Also why have the word project included? That is implicit).

So;
If the project manger wants buildings, the comment could be "#hotosm12345 Mapping buildings"
If the project manger wants roads, the comment could be "#hotosm12345 Mapping roads"
If the project manger wants building and roads, the comment could be "#hotosm12345 Mapping buildings roads"

This does not mean the project manager cannot add more detail, but as the change set comment is limited to 255 characters this should not be so wordy that it prevents mappers from adding more detail if they wish.

**Comment by Patrik-Br** [comment]
Great idea Adrian!

**Comment by RAytoun** [comment]
Would it be possible to automate the changeset comment to add a hashtag **validation** when a task is opened for validation rather than adding it manually. It can help us to assess the amount of fixing the validators do for a project.

**Comment by Adrian-Shobrooke** [comment]
> Would it be possible to automate the changeset comment to add a hashtag **validation** when a task is opened for validation rather than adding it manually. It can help us to assess the amount of fixing the validators do for a project.

Oh, I like that!
